SDK/simUtil/Triton.h.in

```diff
@@ -0,0 +1,40 @@
+/* -*- mode: c++ -*- */
+/****************************************************************************
+ *****                                                                  *****
+ *****                   Classification: UNCLASSIFIED                   *****
+ *****                    Classified By:                                *****
+ *****                    Declassify On:                                *****
+ *****                                                                  *****
+ ****************************************************************************
+ *
+ *
+ * Developed by: Geeks And Nerds Corporation
+ *               11247 South Memorial Parkway
+ *               Huntsville, AL 35758
+ *
+ * License for source code at https://simdis.nrl.navy.mil/License.aspx
+ *
+ * The U.S. Government retains all rights to use, duplicate, distribute,
+ * disclose, or release this software.
+ *
+ */
+
+#ifndef SIMUTIL_TRITON_H
+#define SIMUTIL_TRITON_H
+
+/**
+ * @file Triton.h
+ *
+ * Provides a single header for including Triton ocean model based on system specs.  This header
+ * is generated during compilation/installation of the simUtil library.
+ *
+ */
+
+#cmakedefine TRITON_ENABLE
```

Comment:
I'm not entirely sure this works as written.

There's no configure_file() for Triton.h.in.  There's no matching CMake definition for TRITON_ENABLE or SILVERLINING_ENABLE.

I'm leaning pretty strongly to simplifying by just removing the simUtil.h, then there won't be a need for the extra changes.

Reply:
Sorry, I missed a change.  My working source tree for the changes is not the same that was available to push to github, so I was having to manually copy the changes.  I missed the change to `simUtil/CMakeCache.txt`.  Should be pushed now